### ISSUE TITLE
Bring back support for the .Net SDK

### ISSUE THREAD

**Issue by steskalja** [vmware/vsphere-automation-sdk-.net#30]
## Description
When will vmware bring back the .Net SDK? C# and .Net use are growing and are more in use then GO.

## Environment
- SDK version: 
vSphere Automation SDK 7.0U1

- .NET version: 5.0

- vSphere version: 7.0U2

**Comment by cmills-indochino** [comment]
Yes, this needs to be resurrected.  https://insights.stackoverflow.com/survey/2021#technology-most-popular-technologies

C# at 27.86%, Go at 9.55%
